### PR TITLE
Update BNG datatype & BNG/GeoJSON functions to 5.1

### PR DESCRIPTION
**Pull request by csmith-he**
Overview of changes to bring BNG datatype and BNG <--> GeoJSON functions up to 5.1.x

- Added node and nodeid as input parameters to the validate function
- Added post_save function definition to the two functions
- Specified splitSection variable in bngpoint_to_geojson.py to be an integer

### CODE REVIEW

File: package/extensions/functions/geojson_to_bngpoint_function.py [hunk 1]
```diff
@@ -94,51 +161,52 @@ def save_bngpoint(self, tile, request, is_function_save_method=True):
...
+                        self.config["bng_output_nodegroup"], resourceid=tile.resourceinstance_id, parenttile=tile.parenttile
+                    )
                     new_bng_tile.data[bngnode] = gridref
                     new_bng_tile.save()
 
             return
 
         # Save Tile to cement Parent Tile to subsequent bng_output_nodegroup.
-        #tile.save()
+        # tile.save()
```
Comment (aj-he) [inline package/extensions/functions/geojson_to_bngpoint_function.py]: This can be removed now?

**Review by aj-he — COMMENTED** [review]
All looks good, just minor comment tidy.

File: package/extensions/functions/geojson_to_bngpoint_function.py (hunk 1 above)
Comment (csmith-he) [reply package/extensions/functions/geojson_to_bngpoint_function.py]: Removed in latest commit

**Review by aj-he — APPROVED** [review]
👍

**Pull request merged**